tests/test_models.py

```diff
@@ -46,3 +46,34 @@ def test_people_qs_iterable(self):
     def test_people_qs_count(self):
         qs = People.all()
         self.assertEqual(qs.count(), 15)
+
+class FilmTestCase(BaseStarWarsAPITestCase):
```

Comment:
Credits to the instructors for this. It's almost a 'copy/pasta, find/replace-a' job. Their tests for this project were outstanding. Especially showing us how to use the responses library to mock an api. That and the code for the client.py file are really good resources. Thanks @martinzugnoni and @santiagobasulto .